app/src/main/java/com/cp/tasky/core/navigation/NavGraph.kt

```diff
@@ -0,0 +1,51 @@
+package com.cp.tasky.core.navigation
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import com.cp.tasky.auth.login.presentation.LoginScreenRoot
+
+@Composable
+fun SetUpNavGraph(
+    modifier: Modifier = Modifier,
+    navHostController: NavHostController = rememberNavController(),
+    startDestination: String = Screen.Login.route // Replace with SplashScreen as start destination later
+) {
+
+    NavHost(
+        modifier = modifier,
+        navController = navHostController,
+        startDestination = startDestination
+    ) {
+        composable(route = Screen.Splash.route) {
+            // TODO: Add Splash Screen
+        }
```

Comment:
I highly recommend to not create a separate splash screen. Android already has its own splash screen and that can lead to 2 splash screens that will be shown. Instead, you can use the official splash screen API which also lets you show the splash screen while checking if you're still logged in or not.

I explain it in this video: https://www.youtube.com/watch?v=Loo4i5IrZ4Y